src/Tizen.NUI/src/internal/XamlBinding/RotationTypeConverter.cs

```diff
@@ -22,7 +22,7 @@ public override object ConvertFromInvariantString(string value)
                     if (head.Length == 2)
                     {
                         useDefault = false;
-                        string radianOrDegree = head[0].Trim().ToLowerInvariant();
+                        string radianOrDegree = head[0].Trim().ToUpperInvariant();
```

Comment:
if radianOrDegree is changed as Uppercase then the condition below seems to be changed accordingly.
isn't it like `if (radianOrDegree == "D" || "DEGREE"` 
I am not sure so could you check further?
thank you.

Reply:
Thank you. I changed it.